src/Components/Comments.jsx

```diff
@@ -0,0 +1,58 @@
+import { useEffect, useState } from "react";
+import getCommentsById from "../../utils/get-comments-by-id";
+import { useParams } from "react-router-dom";
+
+function Comments() {
+  const { articleId } = useParams();
+  const [comments, setComments] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    getCommentsById(articleId).then((response) => {
+      setIsLoading(false);
+      setComments(response);
+    });
+  }, [articleId]);
+
+  if (isLoading) {
+    return <p>loading...</p>;
+  }
+
+  <div className="comments-container">
```

Comment:
This div isn't being rendered anywhere at all, I can't see why it is here or its purpose.